src/tracks/InstrumentTrack.cpp

```diff
@@ -149,6 +151,10 @@ InstrumentTrack::~InstrumentTrack()
 
 	// now we're save deleting the instrument
 	if( m_instrument ) delete m_instrument;
+
+	if (m_noGroove != NULL) {
+		delete m_noGroove;
```

Comment:
FWIW it's valid to `delete` a `NULL` pointer (it just doesn't do anything). Putting this behind a conditional is superfluous.